homeassistant/components/freebox/router.py

```diff
@@ -50,37 +50,41 @@ async def get_api(hass: HomeAssistant, host: str) -> Freepybox:
 class FreeboxRouter:
     """Representation of a Freebox router."""
 
+    _api: Freepybox
+    mac: str
+
     def __init__(self, hass: HomeAssistant, entry: ConfigEntry) -> None:
         """Initialize a Freebox router."""
         self.hass = hass
         self._entry = entry
         self._host = entry.data[CONF_HOST]
         self._port = entry.data[CONF_PORT]
 
-        self._api: Freepybox = None
+        self._api_open = False
```

Comment:
This is new, added for the `close` routine.
I felt it made it cleaner than checking against `self._api is not None`